packages/sdks-tests/src/e2e-tests/editing.spec.ts

```diff
@@ -455,4 +456,92 @@ test.describe('Visual Editing', () => {
       await page.frameLocator('iframe').getByText('Bye').waitFor();
     });
   });
+
+  test.describe('New Block addition and deletion', () => {
+    test('should add new block below the last block', async ({
+      page,
+      basePort,
+      sdk,
+      packageName,
+    }) => {
+      test.skip(checkIsGen1React(sdk));
+      test.skip(packageName === 'nextjs-sdk-next-app');
+      test.skip(
+        packageName === 'vue',
+        `Failing on the CI: TypeError: Cannot read properties of null (reading 'namespaceURI')`
+      );
```

Comment:
this works on local
![Screenshot 2025-02-28 at 3 16 36 PM](https://github.com/user-attachments/assets/9e90bd78-939e-4033-be5f-6d0ef7f17713)

Reply:
![Screenshot 2025-02-28 at 3 17 15 PM](https://github.com/user-attachments/assets/09d9786b-344c-450e-bbc7-cb1dce4461d5)